### PR TITLE
Test Build 3 for issue-560 - DO NOT MERGE

### PR DESCRIPTION
**Pull request by twhetzel**
QC / build PR for:
- #565

Resolves #ISSUE(s).

## Overview

This PR:
- One
- Two
- Three

### Documentation
Was the documentation added/updated under `docs/`?
- [ ] Yes
- [x] No, updates to the docs were not necessary after careful consideration

### QC
Was the full pipeline run before submitting this PR using `sh run.sh make build-mondo-ingest` on this branch (after 
`docker pull obolibrary/odkfull:dev`), and no errors occurred?
- [ ] Yes
- [ ] No, there are no functional (code-related) changes to the pipeline in the PR, so no re-run is necessary
   
### New Packages
Were any new *Python packages* added?
- [ ] Yes, [requirements.txt.full](https://github.com/INCATools/ontology-development-kit/blob/master/requirements.txt.full) was updated and an [ODK issue](https://github.com/INCATools/ontology-development-kit/issues/new) submitted
- [x] No

Were any other *non-Python packages* added?
- [ ] Yes, [Dockerfile](https://github.com/INCATools/ontology-development-kit/blob/master/Dockerfile) updated and an [ODK issue](https://github.com/INCATools/ontology-development-kit/issues/new) submitted
- [x] No

### PR Review and Conversations Resolved
Has the PR been sufficiently reviewed by at least 1 team member of the Mondo Technical team and all threads resolved?
- [ ] Yes

### CODE REVIEW

File: docs/metrics/doid.md [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
-**Version IRI:** http://purl.obolibrary.org/obo/mondo/sources/2024-06-05/doid.owl
+**Version IRI:** http://purl.obolibrary.org/obo/mondo/sources/2024-06-14/doid.owl
 
 ### Entities and axioms
 
 | Metric | Value |
 | ------ | ----- |
 | Annotation properties | 27 |
-| Axioms | 110948 |
+| Axioms | 116206 |
```
Comment (matentzn) [inline docs/metrics/doid.md:12]: We now have more axioms which seems to be correct

File: src/mappings/doid.sssom.tsv
Comment (matentzn) [inline src/mappings/doid.sssom.tsv:1]:
```
DOID:0050120	hemophagocytic lymphohistiocytosis	skos:exactMatch	OMIMPS:267700	semapv:UnspecifiedMatching
DOID:0050156	interstitial lung disease 2	skos:exactMatch	OMIM:178500	semapv:UnspecifiedMatching
DOID:0050156	interstitial lung disease 2	oboInOwl:hasDbXref	OMIM:178500	semapv:UnspecifiedMatching
```

Everything correct, yay!

File: src/ontology/lexmatch/split-mapping-set/mondo_closematch_doid.tsv
Comment (matentzn) [inline src/ontology/lexmatch/split-mapping-set/mondo_closematch_doid.tsv:1]: such an irrelevant file.

Comment (matentzn) [reply src/ontology/lexmatch/split-mapping-set/mondo_closematch_doid.tsv:1]: (not related to your PR, just musing to myself)

File: src/sparql/fix_mim.ru
Comment (matentzn) [inline src/sparql/fix_mim.ru:1]: this is here because you are merging to `main`, not to the feature branch (no problem, just pointing out).

**Review by matentzn — APPROVED** [review]
Fantastic! Some comments FYI, but all good.

File: src/sparql/fix_mim.ru
Comment (twhetzel) [reply src/sparql/fix_mim.ru:1]: Gotcha. Was not intended, but switched now to feature branch the build came from just for sanity sake even though this does not get merged.

File: src/mappings/doid.sssom.tsv
Comment (twhetzel) [reply src/mappings/doid.sssom.tsv:1]: I also re-checked all examples from the [previous Build](https://github.com/monarch-initiative/mondo-ingest/pull/556/files#r1628997300) were there were issues and confirmed in those examples all lines removed are now included.

Comment (joeflack4) [reply src/mappings/doid.sssom.tsv:1]: Personally I'm fine w/out the diff! That number being so close is convincing enough to me that this addresses the issue.

File: src/sparql/fix_mim.ru
Comment (joeflack4) [inline src/sparql/fix_mim.ru:1]: ## Strange; shouldn't show certain changes
### `fix_mim.ru` shouldn't be added
Your base for this PR is the `issue-560` branch, which already includes this file, so I'm not sure why it's showing as new here too.

And nothing shows up when I do `git diff origin/issue-560-BUILD-3 origin/issue-560 --name-only | grep fix_mim`.

### `fix_make_omim_exact.ru` and `fix_omimps.ru` shouldn't have any changes
There's no diff between these files and your other branch either.

---

So, looks like a GitHub display error for these.

File: src/mappings/omim.sssom.tsv [hunk 2]
```diff
@@ -7826,6 +7825,7 @@ MONDO:0015855		skos:exactMatch	omim.ps:113700	Breasts and/or nipples, aplasia or
 MONDO:0015912		skos:exactMatch	OMIM:155100	macrothrombocytopenia and granulocyte inclusions with or without nephritis or sensorineural hearing loss	semapv:UnspecifiedMatching
 MONDO:0015942		skos:exactMatch	omim.ps:305620	Frontometaphyseal dysplasia	semapv:UnspecifiedMatching
 MONDO:0015977		skos:exactMatch	omim.ps:601495	Agammaglobulinemia	semapv:UnspecifiedMatching
+MONDO:0015991		skos:exactMatch	omim.ps:215700	Citrullinemia	semapv:UnspecifiedMatching
```
Comment (joeflack4) [inline src/mappings/omim.sssom.tsv:7828]: ## Tangent: Why `omim.ps` CURIE?
@twhetzel I see that this is what you were talking about at our meeting earlier.

@matentzn This doesn't seem related to what Trish did. When I do a global search, this file is the only one in `mondo-ingest` with `omim.ps`. So I take it that this is coming from `mondo`? Should we create an issue to address this there?

- [ ] Consider making issue to address `omim.ps`

Comment (joeflack4) [inline src/mappings/omim.sssom.tsv:7828]: ## _Outdated_: Strange; shouldn't show certain changes

I noticed some strangeness, but before I could comment, I had to refresh, and now the issue is gone, so no worries.

<details><summary>Original comments</summary>
<p>

### `fix_mim.ru` shouldn't be added
Your base for this PR is the `issue-560` branch, which already includes this file, so I'm not sure why it's showing as new here too.

And nothing shows up when I do `git diff origin/issue-560-BUILD-3 origin/issue-560 --name-only | grep fix_mim`.

### `fix_make_omim_exact.ru` and `fix_omimps.ru` shouldn't have any changes
There's no diff between these files and your other branch either.

---

So, looks like a GitHub display error for these.

</p>
</details>

Comment (joeflack4) [inline src/mappings/omim.sssom.tsv:7828]: ## Approve
This passes the SOP checks and my general face checking!

**Pull request closed without merging**